stackit/internal/services/argus/argus_acc_test.go

```diff
@@ -439,8 +640,7 @@ func TestAccResource(t *testing.T) {
 					resource.TestCheckResourceAttr("data.stackit_argus_scrapeconfig.scrapeconfig", "saml2.enable_url_parameters", scrapeConfigResource["saml2_enable_url_parameters"]),
 				),
 			},
-
-			// Import
+			// Import 1
```

Comment:
And is the import importing the instance from the last step? The that one does not have alert configs, right?

Reply:
It is importing from the last step, but the last step has alert configs. It doesn't have the emailConfigs because those create non-empty plans and I wanted to minimize the usage of `ExpectNonEmptyPlan`

Reply:
The last test before importing is the `Data source` one, which has alert configs